scripts/plant_growth_form.py

```diff
@@ -0,0 +1,101 @@
+#retriever
+from pkg_resources import parse_version
+
+from retriever.lib.models import Table, Cleanup, correct_invalid_value
+from retriever.lib.templates import Script
+
+try:
+    from retriever.lib.defaults import VERSION
+
+    try:
+        from retriever.lib.tools import open_fr, open_fw, to_str
+    except ImportError:
+        from retriever.lib.scripts import open_fr, open_fw
+except ImportError:
+    from retriever import open_fr, open_fw, VERSION
+
+
+class main(Script):
+    def __init__(self, **kwargs):
+        Script.__init__(self, **kwargs)
+        self.name = "plant-growth-form"
+        self.title = "Plant Growth Form"
+        self.ref = "http://onlinelibrary.wiley.com/doi/10.1002/ecy.1569/full"
+        self.urls = {
+            "plantgrowth": "http://onlinelibrary.wiley.com/store/10.1002/ecy.1569/asset/supinfo/ecy1569-sup-0001-DataS1.zip?v=1&s=219c74fa85bfe6738e649bedd844f40428351802"}
+        self.citation = "Engemann, K., Sandel, B., Boyle, B. L., Enquist, B. J., Jorgensen, P. M.," \
+                        "Kattge, J., McGill, B. J., Morueta-Holme, N., Peet, R. K., Spencer, N. J., Violle, C., Wiser," \ 
```

Comment:
There is an extra space at the end of line 27 after the "\" that is causing the script not to compile.